pyopenms_viz/_matplotlib/core.py

```
@@ -788,3 +796,40 @@ def _add_box_boundaries(self, annotation_data):
     # since matplotlib is not interactive cannot implement the following methods
     def get_manual_bounding_box_coords(self):
         pass
+
+@APPEND_PLOT_DOC
+class MATPLOTLIBSpectrumPlot(MATPLOTLIB_MSPlot, SpectrumPlot):
```

Ruff (0.8.2)

Redefinition of unused `MATPLOTLIBSpectrumPlot` from line 589

(F811)